atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -68,3 +71,202 @@ export const fatalityGridTableColumns = {
+            },
+          ],
+        },
+      },
+      {
+        id: "status_open_long_term",
+        label: "Open - Long Term",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 2 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_rec",
+        label: "Closed - No Action Recommended",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 3 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_rec_implemented",
+        label: "Closed - Recommendation Implemented",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 4 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_taken",
+        label: "Closed - No Action Taken",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 5 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_null",
+        label: "Null",
+        filter: {
+          where: [
+            {
+              "_not: { recommendation: {recommendation_status_id: { _is_null: false } } }": null,
+            },
+          ],
+        },
+      },
+    ],
+  },
+  groupRoad: {
+    icon: "road",
+    label: "Roadway System",
+    filters: [
+      {
+        id: "on_system",
+        label: "On-System",
+        filter: {
+          where: [
+            {
+              'crash: { onsys_fl: { _eq: "Y" } }': null,
+            },
+          ],
+        },
+      },
+      {
+        id: "off_system",
+        label: "Off-System",
+        filter: {
+          where: [
+            {
+              'crash: { onsys_fl: { _eq: "N" } }': null,
+            },
+          ],
+        },
+      },
+      {
+        id: "road_system_null",
+        label: "Null",
+        filter: {
+          where: [
+            {
+              "_not: { crash: { onsys_fl: { _is_null: false } } }": null,
+            },
+          ],
+        },
+      },
+    ],
+  },
+};
+
+export const fatalityExportFields = `
+crash_id
+person_id
+primaryperson_id
+victim_name
+year
+crash_date
+crash_time
+location
+ytd_fatality
+ytd_fatal_crash
+law_enforcement_num
+case_id
+recommendation { rec_text }
+recommendation { rec_update }
+recommendation { atd__recommendation_status_lkp { rec_status_desc } }
+recommendation { recommendations_partners { atd__coordination_partners_lkp { coord_partner_desc }} }
+primaryperson { unit { unit_description { veh_unit_desc_desc } } }
+person { unit { unit_description { veh_unit_desc_desc } } }
+primaryperson { unit { body_style { veh_body_styl_desc } } }
+person { unit { body_style { veh_body_styl_desc } } }
+crash { onsys_fl }
```

Comment:
i chose these export fields based on what i assumed the vz team would expect to be able to export bc they didnt give an explicit list. since they wanted toggles for unit types i figured unit type and body style would be good fields to carry over into a csv export.